content/learn/quickstart.adoc

```diff
@@ -0,0 +1,89 @@
+---
+layout: default
+title: Patterns quick start
+menu: learn
+weight: 20
+---
+
+== Patterns quick start
+
+Each pattern can be deployed using the command line. The only requirement is to have 
+`git` and `podman` installed. See the <<installation_prerequisites>> for more information.
+
+Check the `values-\*.yaml` for changes that are needed before deployment. After
+changing the `values-*.yaml` files where needed and pushing them to your git
+repository, you can run `./pattern.sh make install` from your local repository directory and
+that will deploy the datacenter/hub cluster for a pattern. Edge clusters are
+deployed by joining/importing them into ACM on the hub.
+
+Alternatively to the `./pattern.sh make install` method, you can use the
+https://operatorhub.io/operator/patterns-operator[validated pattern
+operator] available in the OpenShift console.
+
+For information on using the Validated Patterns Operator please see
```

Comment:
See IBM Style guidance for [please](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#please).
```suggestion
For information on using the Validated Patterns Operator, see
```

Reply:
Ack